src/atoms/fieldset/Fieldset.html

```diff
@@ -1,13 +1,9 @@
 <fieldset
```

Comment:
I think fieldset should have `:class="getClass` even if we don't add any classes on alpaca level so that they can be easily added in config without overriding the template